db/merge_helper.h

```diff
@@ -78,7 +78,7 @@ class MergeHelper {
   //
   // REQUIRED: The first key in the input is not corrupted.
   Status MergeUntil(InternalIterator* iter,
-                    RangeDelAggregator* range_del_agg = nullptr,
+                    CompactionRangeDelAggregatorV2* range_del_agg = nullptr,
```

Comment:
It might be possible, though I need to put some thought into whether or not I want to expose `StripeRep` (or maybe just the forward iter; still need to think more about this). At any rate, I'll leave a TODO here, since it's not a bad thing to avoid the map lookup here.